### PR TITLE
Updates deduplication ID and group ID max length to 128

### PR DESCRIPTION
**Pull request by nishadcptn**
## Motivation
Fixed https://github.com/localstack/localstack/issues/9678

## Changes
Updated max fifo_id len from 127 to 128 characters

### CODE REVIEW

**Comment by nishadcptn** [comment]
I have read the CLA Document and I hereby sign the CLA

File: localstack/services/sqs/provider.py [hunk 1]
```diff
@@ -429,12 +429,12 @@ def check_attributes(message_attributes: MessageBodyAttributeMap):
...
     if not fifo_id:
         return
-    if len(fifo_id) >= 128:
+    if len(fifo_id) > 128:
         raise InvalidParameterValueException(
-            "Message deduplication ID and group ID must be shorter than 128 bytes"
+            f"Value {fifo_id} for parameter {parameter} is invalid. Reason: {parameter} can only include alphanumeric and punctuation characters. 1 to 128 in length."
         )
     if not re.match(sqs_constants.FIFO_MSG_REGEX, fifo_id):
         raise InvalidParameterValueException(
```
Comment (thrau) [inline localstack/services/sqs/provider.py:440]: maybe something we should clean up in the future

**Review by thrau — APPROVED** [review]
LGTM now. thanks for doing the iteration!

**Comment by thrau** [comment]
and welcome to the list of contributors :-)

File: localstack/services/sqs/provider.py (hunk 1 above)
Comment (nishadcptn) [reply localstack/services/sqs/provider.py:440]: Thanks for your review

**Pull request merged**